tables/hbo.tbl

```diff
@@ -0,0 +1,208 @@
+
+
+#The first Hebrew Braille table hosted on Lib Louis was developed by the Library for the Blind and the Ministry of Education in Israel. That  table is based on the IHBC which was developed in the mid-1930’s. Please see World Braille usage 3rd edition, p. 74. 
+#However, Classical or Biblical Hebrew, which is used in the Hebrew Bible and other liturgical literature, contains cantillation marks that are not supported by the Hebrew table mentioned above. This table seeks to provide access to these accents. This table is   also based upon the tables mentioned above.  This table only departs from it where the accents are concerned. With this table, the user will have access to the accents and Masorah in BHS and BHS Quinta. For more information about the IHBC and How it was developed:
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#-maintainer: Paul Geoghegan <contact@envisionly.tech>
+#-maintainer: Matityhau Yeshurun <yeatersink@gmail.com>
+
+# always op code characters
+always ִי 35  # hiriq-yod
+always ֵי 3456  # tsere-yod
+always ּו 136  # shuruq
+always ּוְ 136-3  # shuruq sheva
+always בְּ 12-3  # bet with dagesh sheva
+always בּ 12  # bet with dagesh
+always דּ 5-145  # dalet with dagesh
+always דְּ 5-145-3  # dalet with dagesh sheva
+always וֹ 246  # holam-vav
+always וּ 136  # shuruq
+always יּ 5-245  # Yod with dagesh
+always יְּ 5-245-3  # Yod with dagesh sheva
+always כּ 13  # kaf with dagesh
+always כְּ 13-3  # kaf with dagesh sheva
+always לּ 5-123  # lamed with dagesh
+always לְּ 5-123-3  # lamed with dagesh sheva
+always מּ 5-134  # mem with dagesh
+always מְּ 5-134-3  # mem with dagesh sheva
+always נּ 5-1345  # nun with dagesh
+always נְּ 5-1345-3  # nun with  dagesh sheva
+always פּ 5-1234  # pe with dagesh
+always פְּ 5-1234-3  # pe with dagesh sheva
+always קּ 5-12345  # qof with dagesh
+always קְּ 5-12345-3  # qof with dagesh sheva
+always שְׁ 146-3  #  Shin with dot and sheva
+always שְׂ 156-3  #  Sin sheva
+always שׁ 146  # shin with dot
+always שׂ 156  # sin
+always ת 1456  # tav
+always תּ 2345  # tav with dagesh
+always תְּ 2345-3  # tav with dagesh sheva
+always בּ 12  # Bet with dagesh
+always בְּ 12-3  # Bet with dagesh sheva
+always כּ 13  # KAF with dagesh 
+always כְּ 13-3  # KAF with dagesh  sheva
+always תּ 5-1456  # tav with daggesh
+always תְּ 5-1456-3  # tav with daggesh sheva
+
+# letter op code characters
+letter ֒ 46-234  # segol - isolated
+letter ֓ 46-146  # shalshelet
+letter ֔ 2  # zaqef qatan
+letter ֕ 46-1245  # zaqef gadol
+letter ֖ 46-2345  # tipeha
+letter ֗ 46-1235  # revia
+letter ֘ 46-145  # zarqa
+letter ֙ 46-34  # pashta
+letter ֚ 46-245  # yetiv
+letter ֛ 46-1236  # tevir
+letter ֜ 46-1  # geresh
+letter ֝ 46-16  # geresh muqdam
+letter ֞ 46-12  # gershayim
```

Comment:
ok so now I'm glad I kept looking at this. I've just noticed that both geresh and gershayim are defined multiple times (and differently!) in different parts of this file. 

on line 90, geresh is mapped to 46-1, and then on the next line, geresh muqdam is mapped to 46-16, and gershayim is 46-12.  but then later in the file, you've got geresh as 456-126 and gershayim as 456. 

I don't know what the difference is between a geresh and a geresh muqdam, and googling it leads me to believe that they are the same thing, or maybe that geresh muqdam is what you say when you mean "the geresh that is used as a cantillation mark" as opposed to "the geresh that is used to indicate loan words in modern Hebrew".  But that is a remark about where it is used, not how it is encoded. It looks like there is a separate unicode symbol called geresh muqdam (U059D) which is different from geresh (U05F3). But I'm getting the sense that geresh, geresh muqdam, and apostrophe are probably all used more or less interchangeably by sighted typists, and that a sighted person would not be able to tell the difference between these three characters if they were not examining the underlying unicode. I don't think it makes sense to have different braille symbols for characters that are visually identical, which geresh and geresh muqdam seem to be. 

This confusion implies to me that it would be a good idea to include a test case in your spec file that shows how the geresh and gershayim might appear in a text you might be reading, and what the braille output you'd expect would be.